mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -569,10 +739,11 @@
         <Food value="6"/>
         <Collide index="Locust" value="1"/>
         <Collide index="Phased" value="1"/>
-        <SubgroupPriority value="32"/>
+        <SubgroupPriority value="28"/>
```

Comment:
**undocumented**: Hatchery's subgroup priority changed from 32 to 28

This changes the priority of the wireframe or unit button in several UI elements, most notably the multi-unit-selection panel showing all the wireframes